Error while starting a storybook from default configuration

## Context

I can't make a storybook default configuration work by running `yarn storybook` command.

## Expected Behavior

Storybook should start

## Current Behavior

The build is failing at 99% with the crash report listed in section below

## Failure logs

<details>
  <summary>Full logs</summary>

  ```
  "C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\yarn\bin\yarn.js" run storybook
yarn run v1.22.17
$ start-storybook -p 6006
info @storybook/react v6.4.9
info
info => Loading presets
info => Using implicit CSS loaders
info => Using default Webpack5 setup
<i> [webpack-dev-middleware] wait until bundle finished
10% building 0/16 entries 11/17 dependencies 0/11 modules
info => Ignoring cached manager due to change in manager config
<i> [webpack-dev-middleware] wait until bundle finished
16% building 2/16 entries 2700/2842 dependencies 331/580 modules
warn - The `purge`/`content` options have changed in Tailwind CSS v3.0.
warn - Update your configuration file to eliminate this warning.
99% done plugins webpack-hot-middlewarewebpack built preview 6379bfa5f50c62e53848 in 16879ms
ModuleNotFoundError: Module not found: Error: Can't resolve '..\__mocks__\webextension-polyfill-ts.ts' in 'X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\src\popup'
Did you mean './..\__mocks__\webextension-polyfill-ts.ts'?
Requests that should resolve in the current directory need to start with './'.
Requests that start with a name are treated as module requests and resolve within module directories (node_modules).
If changing the source code is not an option there is also a resolve options called 'preferRelative' which tries to resolve these kind of requests in the current directory too.
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\Compilation.js:2011:28
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:795:13
    at eval (eval at create (X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:10:1)
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:275:22
    at eval (eval at create (X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:9:1)
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:431:22
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:124:11
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:667:25
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:852:8
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:972:5
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\neo-async\async.js:6883:13
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\lib\NormalModuleFactory.js:959:8
    at finishResolved (X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:278:11)
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:342:25
    at X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\node_modules\enhanced-resolve\lib\Resolver.js:409:24
    at eval (eval at create (X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules\webpack\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:27:1)
resolve '..\__mocks__\webextension-polyfill-ts.ts' in 'X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\src\popup'
  Parsed request is a module
  using description file: X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\package.json (relative path: ./src/popup)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\src\popup\node_modules doesn't exist or is not a directory
      X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\src\node_modules doesn't exist or is not a directory
      looking for modules in X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\node_modules
        single file module
          using description file: X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\package.json (relative path: ./__mocks__/webextension-polyfill-ts.ts)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\__mocks__\webextension-polyfill-ts.ts doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\__mocks__\webextension-polyfill-ts.ts.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\__mocks__\webextension-polyfill-ts.ts.tsx doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\__mocks__\webextension-polyfill-ts.ts.js doesn't exist
        X:\AwesomeProjects\aeksco\react-typescript-web-extension-starter\__mocks__\webextension-polyfill-ts.ts doesn't exist
      X:\AwesomeProjects\aeksco\node_modules doesn't exist or is not a directory
      X:\AwesomeProjects\node_modules doesn't exist or is not a directory
      X:\node_modules doesn't exist or is not a directory

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Process finished with exit code 1



  ```
</details>

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Download project on the local machine
2. Install dependencies with `yarn`
3. Run command `yarn storybook`


## Specs

I'm using Windows 10 with node version 14.15.0

--- a/.storybook/main.js
+++ b/.storybook/main.js
@@ -48,7 +48,16 @@ module.exports = {
                     );
 
                     // Updates the `resource.request` to reference our mocked module instead of the real one
-                    resource.request = relativePath;
+                    switch (process.platform) {
+                        case "win32": {
+                            resource.request = "./" + relativePath;
+                            break;
+                        }
+                        default: {
+                            resource.request = relativePath;
+                            break;
+                        }
+                    }
                 },
             ),
         ];

